client/signup/steps/theme-selection/theme-thumbnail.jsx

```diff
@@ -7,14 +7,18 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var analytics = require( 'analytics' ),
-	SignupActions = require( 'lib/signup/actions' ),
-	ThemeHelper = require( 'lib/themes/helpers' );
+	SignupActions = require( 'lib/signup/actions' );
 
 module.exports = React.createClass( {
 	displayName: 'ThemeThumbnail',
 
+	propTypes: {
+		themeName: React.PropTypes.string.isRequired,
+		themeSlug: React.PropTypes.string.isRequired,
+	},
```

Comment:
Why not just pass `theme` as the only prop? It'd make it more consistent with the changes in other files, and we could replace the above with

```js
{
  theme: PropTypes.Shape( {
    name: PropTypes.string.isRequired,
    slug: PropTypes.string.isRequired
  } ).isRequired
}